client/app/modules/series/root/DetailSerieSF.js

```diff
@@ -0,0 +1,26 @@
+import React, { Component, Fragment } from 'react';
+import { connect } from 'react-redux';
+import { getSerieRequest } from 'state/series/actions/getSerie'
+
+class DetailSerieSF extends Component {
+
+    componentDidMount() {
+        this.props.getSerie(this.props.match.params.id)
+    }
+
+    render() {
+        return (
+            <Fragment>
+                { this.props.render(this.props) }
+            </Fragment>
+        )
+    }
+}
+
+const mapStateToProps = ({ serie }) => ({ serie });
+
+const mapDispatchToProps = dispatch => ({
+    getSerie: id => dispatch(getSerieRequest(id)),
+});
+
+export default  connect(mapStateToProps, mapDispatchToProps)(DetailSerieSF)
```

Comment:
Remove space before connect. Another question, are you using the trailing semicolon or not? Don´t be ambiguous

Reply:
I try to use ir allways but sometimes i forgot the semicolon.